html-forms/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <div>
+        <label for="email">Email</label>
+        <div>
+          <input type="text" name="email" />
+        </div>
+        <div>Why is coding cool?</div>
```

Comment:
This should be a `label`

Reply:
ok thanks, still not finished... perhaps I should have committed but not pushed? (still trying to get a hold of the whole git thing)